packages/telemed-intake/app/src/components/CancelVisitDialog.tsx

```diff
@@ -47,8 +46,6 @@ export const CancelVisitDialog: FC<CancelVisitDialogProps> = ({ onClose, appoint
         },
       },
     );
-
-    handleClose();
```

Comment:
Removing this line and only closing on onsuccess leads to a better UX, however, I think if we are not closing on error, we should indicate some kind of error message in dialog, setting it onerror. Something like this:

`{error && (
        <Typography color="error" sx={{ mb: 2 }}>
          {cancelVisitErrorMessage}
        </Typography>
      )}`